crypto/fipsmodule/aes/asm/aesni-x86_64.pl

```diff
@@ -1296,76 +1296,6 @@ sub aesni_generate8 {
 	xor	%eax,%eax
 	jmp	.Lenc_key_ret
 
-.align	16
```

Comment:
Add the comment about removing 192-bit key support.

Reply:
Resolved [here](https://github.com/briansmith/ring/pull/707/files#diff-34a7b7b2dc56d68bca3fb994668a2ef9R1299)